docs/pages/extensibility/api.md

```diff
@@ -27,9 +27,9 @@ Currently, Meshery only requires a valid token in order to allow clients to invo
 
 ### Endpoints
 
-Each of the API endpoints are exposed through [server.go](https://github.com/layer5io/meshery/blob/master/router/server.go). Endpoints are grouped by function (e.g. /api/mesh or /api/perf).
+Each of the API endpoints are exposed through [server.go](../../../server/router/server.go). Endpoints are grouped by function (e.g. /api/mesh or /api/perf).
```

Comment:
The link here needs to be an absolute path to the ```server.go``` file, since the link is being rendered outside of the GitHub.